ui/shared/components/panel/variants/selectors.test.js

```diff
@@ -37,19 +37,16 @@ test('getPairedSelectedSavedVariants', () => {
   const savedFamilyVariants = getPairedSelectedSavedVariants(
     STATE_WITH_2_FAMILIES, { match: { params:  { familyGuid: 'F011652_1' } } }
   )
-  expect(savedFamilyVariants.length).toEqual(3)
+  expect(savedFamilyVariants.length).toEqual(2)
   expect(savedFamilyVariants[0].variantGuid).toEqual('SV0000004_116042722_r0390_1000')
   expect(savedFamilyVariants[1].variantGuid).toEqual('SV0000002_1248367227_r0390_100')
-  expect(savedFamilyVariants[2].variantGuid).toEqual('SV0000002_SV48367227_r0390_100')
```

Comment:
in the test fixture data this variant is not associated with any tags or notes, and the new logic does not return it. Since There is no way to return saved variants from the server without tags or notes this difference is moot